strategy/live_trader.py

```
@@ -911,24 +950,81 @@ def get_open_orders_list(self) -> List[Dict[str, Any]]:
+                                if isinstance(created_raw, (int, float)):
+                                    ts_val = created_raw / 1000.0 if created_raw > 1e11 else float(created_raw)
+                                    time_str = time.strftime("%H:%M:%S", time.localtime(ts_val))
+                                elif isinstance(created_raw, str):
+                                    try:
+                                        from datetime import datetime
+                                        iso_str = created_raw.replace("Z", "+00:00")
+                                        dt = datetime.fromisoformat(iso_str)
+                                        if dt.tzinfo is not None:
+                                            dt = dt.astimezone()
+                                        time_str = dt.strftime("%H:%M:%S")
+                                    except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)